recipes/pangwas/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "pangwas" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/phac-nml/pangwas/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: f7484e1b7cd9dbde4e5252f830b67e474dc1a4a4f986e9539ed671240372d608 
+
+build:
+  noarch: generic
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vvv --no-build-isolation --no-deps
+  entry_points:
+    - pangwas = pangwas:cli
+  # If the software version starts with a 0 (e.g. 0.3.2) semantic versioning allows breaking changes in minor releases.
+  # https://bioconda.github.io/contributor/linting.html#missing-run-exports
+  run_exports:
+    - {{ pin_subpackage('pangwas', max_pin='x.x') }}
+
+requirements:
+  host:
+    - python >=3.10,<3.11
```

Comment:
why only for 1 python version?

Reply:
Thank you for the suggestion! Originally because that was the version definition in the package's `pyproject.toml`:

https://github.com/phac-nml/pangwas/blob/main/pyproject.toml#L15

I will work on relaxing that and post back when I've resolved it.

Reply:
I did some digging and that python requirement comes from the dependency `bakta` which requires python<3.11: 
https://github.com/bioconda/bioconda-recipes/blob/35db4ed91bfc1ce88939f5622ab27d9eea4e3bc9/recipes/bakta/meta.yaml#L25

For `pangwas`, upgrading `bakta` will be a significant model change and require extensive testing. So I've
flagged that for my own research in the new release: https://github.com/phac-nml/pangwas/issues/16

With this context, do you find this a satisfactory reason for the `python` dependency lock to `3.10`?